### PR TITLE
[Windows] Install Docker-wincred

### PR DESCRIPTION
**Pull request by maxim-lobanov**
# Description
It is official docker helper provided by docker team: https://github.com/docker/docker-credential-helpers
Currently, we install this tool in provisioner. This PR moves this logic to image generation code.

## Check list
- [x] Tests are written (if applicable)
- [x] Documentation is updated (if applicable)
- [x] Changes are tested and related VM images are successfully generated

### CODE REVIEW

**Review by miketimofeev — APPROVED** [review]

**Review by dibir-magomedsaygitov — APPROVED** [review]

**Review by al-cheb — APPROVED** [review]

File: images/win/scripts/Installers/Install-Docker.ps1 [hunk 1]
```diff
@@ -14,6 +14,13 @@ Start-Service docker
 Write-Host "Install-Package Docker-Compose"
 Choco-Install -PackageName docker-compose
 
+Write-Host "Install docker-wincred"
+$dockerCredLatestRelease = Invoke-RestMethod -Uri "https://api.github.com/repos/docker/docker-credential-helpers/releases/latest"
+$dockerCredDownloadUrl = $dockerCredLatestRelease.assets.browser_download_url -match "docker-credential-wincred-.+\.zip" | Select-Object -First 1
+$dockerCredArchive = Start-DownloadWithRetry -Url $dockerCredDownloadUrl
+Expand-Archive -Path $dockerCredArchive -DestinationPath "C:\Program Files\Docker"
```
Comment (shilovmaksim) [inline images/win/scripts/Installers/Install-Docker.ps1:21]: Maybe we should use [Extract-7Zip](https://github.com/actions/virtual-environments/blob/f38b2174698750ed6d8e5297458c0df698958345/images/win/scripts/ImageHelpers/InstallHelpers.ps1#L418) here? Which is considered to be a little bit faster and stable.

Comment (shilovmaksim) [reply images/win/scripts/Installers/Install-Docker.ps1:21]: Oh, just realized that 7-zip not installed yet.

**Review by shilovmaksim — APPROVED** [review]

File: images/win/scripts/Installers/Install-Docker.ps1 (hunk 1 above)
Comment (maxim-lobanov) [reply images/win/scripts/Installers/Install-Docker.ps1:21]: Yep,  7zip is installed later. I don't want to change order of scripts in this PR.
actually, archive size is 1.3 MB so it shouldn't be a problem

**Review by AlenaSviridenko — APPROVED** [review]

**Pull request merged**